docs/apache-airflow/extra-packages-ref.rst

```diff
@@ -208,6 +208,8 @@ Here's the list of the :ref:`subpackages <installation:extra_packages>` and what
 +=====================+=====================================================+======================================================================+
 | cgroups             | ``pip install 'apache-airflow[cgroups]'``           | Needed To use CgroupTaskRunner                                       |
 +---------------------+-----------------------------------------------------+----------------------------------------------------------------------+
+| crypto              | ``pip install 'apache-airflow[crypto]'``            | Used for cryptography operations                                     |
```

Comment:
I think it would be useful to know that this option is deprecated and not needed in Airflow 2.0

Reply:
I 've already added and issue for that: https://github.com/apache/airflow/issues/12851. Let'\s move the conversation there if you have some ideas how to do it.

Reply:
I also modified slightly the pre-commit now to be able to remove crypto from the check, and I indeed left an "empty' crypto requirement.

I also added a new table of deprecated extras. in the doc.